src/plan/generational/global.rs

```diff
@@ -74,6 +74,9 @@ impl<VM: VMBinding> CommonGenPlan<VM> {
     /// Release Gen. This should be called by a single thread in GC release work.
     pub fn release(&mut self, tls: VMWorkerThread) {
         let full_heap = !self.is_current_gc_nursery();
+        if VM::VMObjectModel::GLOBAL_LOG_BIT_SPEC.is_on_side() {
+            self.nursery.clear_side_log_bits();
```

Comment:
Why do we need to clear the log bits in the nursery? The nursery in a generational plan should never have log bits set. If you're saying that plans need to inform spaces how to maintain log bits, then I'm not sure why this is necessary since the plan knows that the log bits are never set for STW generational plans.

Reply:
I agree it is unnecessary, I will remove it.

I added this code because we call `self.nursery.release()`, and there used to be side log bits clearing in `CopySpace::release`.